### PR TITLE
Add Miles' Flower Braid

### PR DESCRIPTION
**Pull request by FlannelVioiln**
## About the PR
Added Mile's Flower Braid.
Will really only work on the Braid-Tail hair style. Otherwise its just floating flowers.

## Why / Balance
Wanted to add a personal item for Georgia A Miles into umbra

## Technical details
just added some pngs

## Media
image
image
image
image
image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
none

**Changelog**
:cl:
- add: Added Mile's Flower Braid

### CODE REVIEW

**Comment by Tych0theSynth** [comment]
You can't add this directly into the master files/directories I'm afraid. They'll need to go in our specific _Umbra directories.
The sprites will need to go into:
/Textures/_Umbra/Clothing/Head

and YAML will need to go into (you will need to create it front scratch):
/Prototypes/_Umbra/Entities/Clothing/Head

(You will need to create the head/hats directory if it doesn't exist yet)

File: Resources/Prototypes/Entities/Clothing/Head/misc.yml [hunk 1]
```diff
@@ -50,6 +50,19 @@
...
+  name: Mile's flower braid
+  description: Georgia's braid is woven with tiger lillies and vines.
+  components:
+  - type: Sprite
+    sprite: Clothing/Head/Misc/georgia-flower-wreath.rsi
+  - type: Clothing
+    sprite: Clothing/Head/Misc/georgia-flower-wreath.rsi
+    slots:
+    - HEAD
+
```
Comment (TsjipTsjip) [inline Resources/Prototypes/Entities/Clothing/Head/misc.yml:65]: Please add a `suffix: PersonalItem` line, so we can easily identify these from other items.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
Minor remarks, see also Tych0's remark above. If needed we can walk you through this.

**Comment by FlannelVioiln** [comment]
> You can't add this directly into the master files/directories I'm afraid. They'll need to go in our specific _Umbra directories. The sprites will need to go into: /Textures/_Umbra/Clothing/Head
> 
> and YAML will need to go into (you will need to create it front scratch): /Prototypes/_Umbra/Entities/Clothing/Head
> 
> (You will need to create the head/hats directory if it doesn't exist yet)

ive created the new directory and moved everything around, but now the sprint is very broken. Am I suppoesd to add a reference somewhere that I'm missing

**Comment by Tych0theSynth** [comment]
> > You can't add this directly into the master files/directories I'm afraid. They'll need to go in our specific _Umbra directories. The sprites will need to go into: /Textures/_Umbra/Clothing/Head
> > and YAML will need to go into (you will need to create it front scratch): /Prototypes/_Umbra/Entities/Clothing/Head
> > (You will need to create the head/hats directory if it doesn't exist yet)
> 
> ive created the new directory and moved everything around, but now the sprint is very broken. Am I suppoesd to add a reference somewhere that I'm missing

Have you checked the meta.json for the sprites to make sure that's correct, and that the files are looking for the sprites via the correct path?

**Comment by arimah** [comment]
> ive created the new directory and moved everything around, but now the sprint is very broken. Am I suppoesd to add a reference somewhere that I'm missing

You don't appear to have pushed the edited code, so I can't comment on that directly. Make sure that the paths on the `Sprite` and `Clothing` components refer to the correct location, including `_Umbra/` as appropriate. You can always look at other personal items as examples. :)

**Comment by FlannelVioiln** [comment]
> > > You can't add this directly into the master files/directories I'm afraid. They'll need to go in our specific _Umbra directories. The sprites will need to go into: /Textures/_Umbra/Clothing/Head
> > > and YAML will need to go into (you will need to create it front scratch): /Prototypes/_Umbra/Entities/Clothing/Head
> > > (You will need to create the head/hats directory if it doesn't exist yet)
> > 
> > 
> > ive created the new directory and moved everything around, but now the sprint is very broken. Am I suppoesd to add a reference somewhere that I'm missing
> 
> Have you checked the meta.json for the sprites to make sure that's correct, and that the files are looking for the sprites via the correct path?

turns out i had the extra an extra /misc/ in the path referenced in the yml. all good and building now.

File: Resources/Textures/Clothing/Head/Misc/georgia-flower-braid.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (arimah) [inline Resources/Textures/Clothing/Head/Misc/georgia-flower-braid.rsi/meta.json:1]: Remove the entire Resources/Textures/Clothing/Head/Misc/georgia-flower-braid.rsi directory, including this file and the sprites in it. The others are in the right place. :) Don't forget to `git add` removals!

**Review by arimah — CHANGES_REQUESTED** [review]

File: Resources/Textures/Clothing/Head/Misc/georgia-flower-braid.rsi/meta.json (hunk 2 above)
Comment (FlannelVioiln) [reply Resources/Textures/Clothing/Head/Misc/georgia-flower-braid.rsi/meta.json]: done!

**Review by TsjipTsjip — APPROVED** [review]
GM approval. Files also look good.

**Review by arimah — APPROVED** [review]

**Pull request merged**